src/master/master.ts

```diff
@@ -81,10 +81,10 @@ export function redactLog(log: LogEntry[], playerID: PlayerID) {
  * Verifies that the game has metadata and is using credentials.
  */
 export const doesGameRequireAuthentication = (
```

Comment:
This will require refactoring the call sites in `master.ts` and `master.test.ts`:

```suggestion
export const doesMatchRequireAuthentication = (
```